test/libsolidity/SemanticTest.cpp

```diff
@@ -212,7 +224,10 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 				);
 			}
 
-			bool outputMismatch = (output != test.call().expectations.rawBytes());
+			bool outputMismatch = output != test.call().expectations.rawBytes();
```

Comment:
I'd keep the parenthesis. I usually add them myself in expressions containing both assignment and boolean or comparison operators for clarity.